commonTools/framework/CMakeLists.txt

```diff
@@ -135,12 +135,13 @@ create_get_changed_trilinos_packages_test(cmake_std_atdm_TeuchosCore_PyTrilinos
 
 create_get_changed_trilinos_packages_test(cmake_std_atdm
   "cmake/std/atdm/anything\ncmake/std/atdm/shiller/tweaks/INTEL-RELEASE-OPENMP-HSW.cmake"
-  ""
-  ""
-  "-- NOTE: No packages are being enabled!"
+  "TrilinosATDMConfigTests"
+  "TrilinosATDMConfigTests"
```

Comment:
Just making sure this isn't a duplicate line.

Reply:
Yes, this is expected.  If you look at how that test function is defined, the first is looking at all of the modified packages while the second is looking at all of the "Primary Tested" modified packages (which are the same in this case).